pkg/virt-api/api.go

```diff
@@ -723,10 +727,15 @@ func (app *virtAPIApp) Compose() {
 }
 
 func (app *virtAPIApp) ConfigureOpenAPIService() {
-	spec := openapi.LoadOpenAPISpec(restful.RegisteredWebServices())
-	config := openapi.CreateOpenAPIConfig(restful.RegisteredWebServices())
+	config := openapi.CreateConfig()
+	config.GetDefinitions = v12.GetOpenAPIDefinitions
+	spec, err := builderv3.BuildOpenAPISpec(restful.RegisteredWebServices(), config)
```

Comment:
nit: usage of deprecated function
```suggestion
        spec, err := builderv3.BuildOpenAPISpecFromRoutes(restfuladapter.AdaptWebServices(restful.RegisteredWebServices()), config)
```

Reply:
Yes good catch, I tried to get away from it, but it is not such an easy task and would require quite a bit more openapi work.

Reply:
IMHO it can be addresses in a follow up PR, to allow this to go forward. Thanks